content/app-dev/create-actor/run.md

```diff
@@ -15,7 +15,7 @@ Building the actor is as easy as running `make` in the project's root directory.
 
 ### Launch the Actor
 
-There are countless ways to run the actor we just created, and if you went through the [getting started](../../../overview/getting-started) section, you'll have seen some some of them already. Open your browser to view the web dashboard [http://localhost:4000](http://localhost:4000) to view the dashboard (you may need to modify the port number`4000` if you changed it before launching the wasmCloud host).
+There are countless ways to run the actor we just created, and if you went through the [getting started](../../../overview/getting-started) section, you'll have seen some some of them already. Open your browser to view the web dashboard [http://localhost:4000](http://localhost:4000) to view the dashboard (you may need to modify the port number `4000` if you changed it before launching the wasmCloud host).
```

Comment:
I know this text is in the original but it reads awkwardly... "Open your browser to view the web dashboard to view the dashboard" ... Maybe make it "Open your browser to view the web dashboard at [http://localhost:4000](http://localhost:4000). If you're using an alternate port number use that here.